doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -0,0 +1,449 @@
+
+   Submit dir: /home/jdoe/tmp/bps/submit/shared/pipecheck/20201111T13h34m08s
+   Run Id: 176261
+
+Adding ``--log-level INFO`` option to the command line outputs more information
+especially for those wanting to watch how long the various submission stages
+take. 
+
+.. _bps-report:
+
+Checking status
+---------------
+
+To check the status of the submitted run, you can use tools provided by
+HTCondor or Pegasus, for example, ``condor_status`` or ``pegasus-status``. To
+get a more pipeline oriented information use
+
+.. code-block:: bash
+
+   bps report
+
+which should display run summary similar to the one below ::
+
+	X      STATE  %S       ID OPERATOR   PRJ   CMPGN    PAYLOAD    RUN                                               
+	-----------------------------------------------------------------------------------------------------------------------
+	     RUNNING   0   176270 jdoe       dev   quick    pcheck     shared_pipecheck_20201111T14h59m26s
+
+To see results regarding past submissions, use ``bps report --hist X``  where
+``X`` is the number of days past day to look at (can be a fraction).  For
+example ::
+
+	$ bps report --hist 1
+		STATE  %S       ID OPERATOR   PRJ   CMPGN    PAYLOAD    RUN                                               
+	-----------------------------------------------------------------------------------------------------------------------
+	   FAILED   0   176263 jdoe       dev   quick    pcheck     shared_pipecheck_20201111T13h51m59s               
+	SUCCEEDED 100   176265 jdoe       dev   quick    pcheck     shared_pipecheck_20201111T13h59m26s               
+
+Use ``bps report --help`` to see all currently supported options.
+
+.. _bps-terminate:
+
+Terminating running jobs
+------------------------
+
+There currently isn’t a BPS command for terminating jobs.  Instead you can use
+the `condor_rm`__ or `pegasus-remove`__.  Both take the ``runId`` printed by
+``bps submit``.  For example
+
+.. code-block:: bash
+
+   condor_rm 176270       # HTCondor
+   pegasus-remove 176270  # Pegasus WMS
+
+``bps report`` also prints the ``runId`` usable by ``condor_rm``.  
+
+If you want to just clobber all of the runs that you have currently submitted,
+you can just do the following no matter if using HTCondor or Pegasus WMS plugin:
+
+.. code-block:: bash
+
+   condor_rm <username>
+
+.. __: https://htcondor.readthedocs.io/en/latest/man-pages/condor_rm.html
+.. __: https://pegasus.isi.edu/documentation/cli-pegasus-remove.php
+
+.. _bps-configuration-file:
+
+BPS configuration file
+----------------------
+
+Configuration file can include other configuration files using
+``includeConfigs`` with YAML array syntax. For example
+
+.. code-block:: YAML
+
+   includeConfigs:
+     - bps-operator.yaml
+     - bps-site-htcondor.yaml
+
+Values in the configuration file can be defined in terms of other values using
+``{key}`` syntax, for example
+
+.. code-block:: YAML
+
+   patch: 69
+   dataQuery: patch = {patch}
```

Comment:
Don't know if these values will catch someone off guard since they aren't in the full example yaml or not.